### DISCUSSION TITLE
custom grid column data format for DateTime object

### DISCUSSION THREAD

**Discussion by Sinkira** [vikramlearning/blazorbootstrap#984] [Q&A] [answered]
Hello, 

I have a grid column for a DateTime and the rows in that column show date as well as time. However, I only want it to show the date. How would this be achieved?

Simply put, I want DateTime to show up as a DateOnly would. 

thanks.

**Comment by gvreddy04** [comment, marked as answer]
@Sinkira 

Here is the example:

```csharp
@context.OrderDate.ToString("MM/dd/yyyy")
```

References:
1. https://demos.blazorbootstrap.com/grid/other#cell-formating
2. https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings
